package.json

```diff
@@ -21,8 +21,8 @@
     "@emotion/styled": "^11.11.0",
     "@mui/icons-material": "^5.15.3",
     "@mui/lab": "^5.0.0-alpha.159",
-    "@mui/material": "^5.15.3",
-    "@mui/styles": "^5.15.3",
+    "@mui/material": "^5.15.15",
+    "@mui/styles": "^5.15.15",
```

Comment:
@karthikbtf this seemed to do the trick. Was just a mismatch in the resolved ThemeProvider between the two packages.